### PR TITLE
Fix subcontainer queries in subcontainers.

### PR DESCRIPTION
**Pull request by vmarmol**
Queries for root were not being correctly diagnosed.

### CODE REVIEW

File: events/handler_test.go [hunk 1]
```diff
@@ -78,6 +80,10 @@ func TestCheckIfIsSubcontainer(t *testing.T) {
 		ContainerName: "/root-completely-different-container",
 	}
 
+	if checkIfIsSubcontainer(rootRequest, sameContainerEvent) {
+		t.Errorf("should have found %v and %v did not have the same container name",
```
Comment (rjnagal) [inline events/handler_test.go]: s/should have/should not have/

File: events/handler_test.go [hunk 2]
```diff
@@ -87,8 +93,13 @@ func TestCheckIfIsSubcontainer(t *testing.T) {
...
 	}
 
+	rootRequest.IncludeSubcontainers = true
 	myRequest.IncludeSubcontainers = true
 
+	if !checkIfIsSubcontainer(rootRequest, sameContainerEvent) {
+		t.Errorf("should have found %v and %v had the same container",
+			rootRequest.ContainerName, sameContainerEvent.ContainerName)
+	}
 	if !checkIfIsSubcontainer(myRequest, sameContainerEvent) {
```
Comment (rjnagal) [inline events/handler_test.go]: I think this check should be checking against subContainerName.

File: events/handler_test.go [hunk 3]
```diff
@@ -87,8 +93,13 @@ func TestCheckIfIsSubcontainer(t *testing.T) {
 			myRequest, subContainerEvent)
 	}
 
+	rootRequest.IncludeSubcontainers = true
 	myRequest.IncludeSubcontainers = true
 
+	if !checkIfIsSubcontainer(rootRequest, sameContainerEvent) {
+		t.Errorf("should have found %v and %v had the same container",
```
Comment (rjnagal) [inline events/handler_test.go]: The error message is hard to follow. Maybe : "should have found %v as %v is a subcontainer."

File: events/handler_test.go (hunk 1 above)
Comment (vmarmol) [inline events/handler_test.go]: Done

File: events/handler_test.go (hunk 3 above)
Comment (vmarmol) [inline events/handler_test.go]: Done

File: events/handler_test.go (hunk 2 above)
Comment (vmarmol) [inline events/handler_test.go]: I *think* this one is fine, we check against subcontainer below it. We check:
- `/root` subcontainerOf `/root` == `true`
- `/root/sub` subcontainerOf `/root` == `true`
- `/root-different` subcontainerOf `/root` == `false`

**Comment by rjnagal** [comment]
LGTM
